packages/@adobe/spectrum-css-temp/components/helptext/index.css

```diff
@@ -48,4 +48,8 @@
     text-transform: var(--spectrum-helptext-neutral-textonly-text-transform);
     letter-spacing: var(--spectrum-helptext-neutral-textonly-text-letter-spacing);
   }
+
+  &.is-disabled {
+    cursor: default;
+  }
```

Comment:
disabled + error help text isn't a supported combo as per https://spectrum.adobe.com/page/help-text/#Disabled so we should probably disable that combo from happening.